doc/api/cli.md

```diff
@@ -726,6 +726,17 @@ Export keyword before a key is ignored:
 export USERNAME="nodejs" # will result in `nodejs` as the value.
 ```
 
+### `--override-env-var`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Override existing environment variables on your machine with values specified in
```

Comment:
Can you expand this with usage from cli flag and runtime function? And also can you add this flag reference to both cli and runtime function documentation?

Reply:
@anonrig I pushed some changes, could you please check again ?

Reply:
Can you also give an example of what it really does? Like actual code example of the behavior. It might not be easily understandable for some.